misc/emacs/juttle-derived-mode.el

```diff
@@ -14,7 +14,7 @@
 ;; (setq juttle-run-dir "/home/<user>/src/juttle/bin")
 
 (defvar juttle-run-dir "")
-(defvar juttle-run-prog "demo")
+(defvar juttle-run-prog "outrigger-client push --topic mydev --path")
```

Comment:
You mean the topic? They can override these in their own config file. If you mean what link they should go to afterward, the output of the program appears in a *compilation* buffer.